WcaOnRails/db/migrate/20171219204656_update_committes_emails.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class UpdateCommittesEmails < ActiveRecord::Migration[5.1]
+  def change
+    execute <<-SQL
+      UPDATE teams set email = 'disciplinary@worldcubeassociation.org' where friendly_id = 'wdc';
+    SQL
+
+    execute <<-SQL
+      UPDATE teams set email = 'regulations@worldcubeassociation.org' where friendly_id = 'wrc';
```

Comment:
Is this also in teams.seeds.rb? If so, we should update that as well (or honestly, we could probably delete that file without it causing issue).

Reply:
LOL, two fails in 24h